src/main/resources/schemas/non-strategic/administrative_court_daily_cause_list.json

```diff
@@ -40,7 +40,7 @@
         "examples": [
           "10am"
         ],
-        "pattern": "^\\d{1,2}([:.]\\d{2})?[ap]m$"
+        "pattern": "^\\d{1,2}([:.]\\d{2})?[ap]m\\s*$"
```

Comment:
There is a parameterized test method testValidateWithErrorWhenInvalidTimeFormat() for most of the list types with time format. I wonder we should add a test case for each of them